provider/doc_edits.go

```diff
@@ -186,6 +186,13 @@ func (r replacementFile) checkForTODOs(path string, content []byte) {
 		start, end = findLine(content, m[0])
 		line := string(content[start:end])
 
+		// Do not add duplicate replacements to requirements.json
```

Comment:
Oof. I didn't dissect his PR into commits and missed this so ended up reimplementing.